php/libraries/NDB_BVL_Feedback.class.inc

```diff
@@ -1002,30 +996,32 @@ class NDB_BVL_Feedback
      *
      * @author Evan McIlroy <evanmcilroy@gmail.com>
      *
-     * @return void
+     * @return int The number of updated threads
      */
     function openThread($feedbackID)
     {
+        if (empty($feedbackID)) {
+            throw new Exception(
```

Comment:
```suggestion
            throw new InvalidArgumentException(
```

Might be nice to be more specific if you leave this here and don't use type hints